2021/Firera/System_monitor_Raspberry/sysMonitor_ARM.py

```diff
@@ -0,0 +1,145 @@
+# SYSTEM RESOURCES MONITOR
+# Launch this script for desired time and terminate the monitoring with ctrl+c
+# point your browser on localhost:8888 to see the Chronograf dashboard
+# join @InfluxdbNotyfy_bot for receiving alerts due resources overload (change chatID value)
+# graphs are saved to ./Analytics/somegraph.png
+
+# IMPORTANT: this program requires InfluxDB, Chronograph and Kapacitor services active (from InfluxData)
+
+from influxdb import InfluxDBClient
+import datetime
+import time
+import os
+from easysnmp import Session
+import webbrowser as web
+import matplotlib.pyplot as pl
+
+
+# snmp constants
+HOSTNAME = 'localhost'
+COMMUNITY = 'public'
+OID_RAM_UNUSED = 'iso.3.6.1.4.1.2021.4.11.0'
+OID_DISK_USAGE = 'iso.3.6.1.4.1.2021.9.1.9.1'
+OID_CPU_USAGE = 'iso.3.6.1.2.1.25.3.3.1.2'
+
+URL_DASHBOARD = 'http://localhost:8888'
+DB_NAME = 'system'
+GRAPHICS_PATH = './Analytics'
+
+# Initializing snmp session
+session = Session(hostname=HOSTNAME, community=COMMUNITY, version=2)
+
+# Initializing database
+client = InfluxDBClient(HOSTNAME, 8086, 'admin', '1234', DB_NAME)
+client.create_database(DB_NAME)
+
+# Open dashboard automatically
+web.open_new_tab(URL_DASHBOARD)
+
+print('Running...')
+print('Press ctrl+c to stop monitoring and view results.')
+
+while True:
+    entry = []   
+    try:
+        payload = {
+                    'measurement' : 'sysMonitor',
+                    'tags': {
+                        'device' : 'raspberry'
+                        },
+                    'time': datetime.datetime.utcnow(),
+                    'fields': {
+                        'availableram' : int(session.get(OID_RAM_UNUSED).value),
+                        'cpuUsage_C1' : int(session.get(OID_CPU_USAGE+'.196608').value),
```

Comment:
Cosa sono questi valori hardcoded? Andrebbero resi parametrici e indipendenti da una CPU specifica come in questo esempio